build.py

```diff
@@ -641,6 +664,22 @@ def build(self):
 
 Project.add(Project_glib_openssl())
 
+class Project_graphene(GitRepo, Meson):
+    def __init__(self):
+        Meson.__init__(self,
+            'graphene',
+            repo_url = 'https://github.com/ebassi/graphene',
+            fetch_submodules = False,
+            tag = None,
+            dependencies = ['ninja', 'meson', 'pkg-config', 'glib'],

```

Comment:
Nearly all meson projects use `dependency()` at some point and the vs2015 backend is only functional for fairly simple projects atm.